eng/DotNetBuild.props

```diff
@@ -1,14 +1,9 @@
 <!-- When altering this file, include @dotnet/product-construction as a reviewer. -->
-
 <Project>
 
   <PropertyGroup>
     <GitHubRepositoryName>diagnostics</GitHubRepositoryName>
     <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
```

Comment:
we could move these as well right?

Reply:
Not yet. This information is used by the ArPow source-build infra to create the intermediate package. That runs in the outer build outside of the repo infra.